[2022-02-21 16:53:23,500] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-31T00:00:00+00:00 [queued]>
[2022-02-21 16:53:23,518] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-31T00:00:00+00:00 [queued]>
[2022-02-21 16:53:23,519] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 16:53:23,519] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-21 16:53:23,519] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 16:53:23,535] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_path> on 2022-01-31 00:00:00+00:00
[2022-02-21 16:53:23,542] {standard_task_runner.py:52} INFO - Started process 120 to run task
[2022-02-21 16:53:23,545] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_path', 'get_path', 'scheduled__2022-01-31T00:00:00+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmprkxfvi1u', '--error-file', '/tmp/tmpvzvgvcin']
[2022-02-21 16:53:23,546] {standard_task_runner.py:77} INFO - Job 34: Subtask get_path
[2022-02-21 16:53:23,617] {logging_mixin.py:109} INFO - Running <TaskInstance: test_path.get_path scheduled__2022-01-31T00:00:00+00:00 [running]> on host 7c24e16e6467
[2022-02-21 16:53:23,678] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_path
AIRFLOW_CTX_TASK_ID=get_path
AIRFLOW_CTX_EXECUTION_DATE=2022-01-31T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-31T00:00:00+00:00
[2022-02-21 16:53:23,679] {logging_mixin.py:109} INFO - /opt/***
[2022-02-21 16:53:23,679] {python.py:175} INFO - Done. Returned value was: None
[2022-02-21 16:53:23,692] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_path, task_id=get_path, execution_date=20220131T000000, start_date=20220221T165323, end_date=20220221T165323
[2022-02-21 16:53:23,721] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-21 16:53:23,754] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-21 17:02:06,041] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-31T00:00:00+00:00 [queued]>
[2022-02-21 17:02:06,056] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-31T00:00:00+00:00 [queued]>
[2022-02-21 17:02:06,056] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 17:02:06,056] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-21 17:02:06,056] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 17:02:06,068] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_path> on 2022-01-31 00:00:00+00:00
[2022-02-21 17:02:06,074] {standard_task_runner.py:52} INFO - Started process 118 to run task
[2022-02-21 17:02:06,078] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_path', 'get_path', 'scheduled__2022-01-31T00:00:00+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmp_rqkvul_', '--error-file', '/tmp/tmpxafhghkt']
[2022-02-21 17:02:06,079] {standard_task_runner.py:77} INFO - Job 32: Subtask get_path
[2022-02-21 17:02:06,142] {logging_mixin.py:109} INFO - Running <TaskInstance: test_path.get_path scheduled__2022-01-31T00:00:00+00:00 [running]> on host ce809d68944a
[2022-02-21 17:02:06,193] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_path
AIRFLOW_CTX_TASK_ID=get_path
AIRFLOW_CTX_EXECUTION_DATE=2022-01-31T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-31T00:00:00+00:00
[2022-02-21 17:02:06,193] {python.py:175} INFO - Done. Returned value was: /nba_database
[2022-02-21 17:02:06,214] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_path, task_id=get_path, execution_date=20220131T000000, start_date=20220221T170206, end_date=20220221T170206
[2022-02-21 17:02:06,249] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-21 17:02:06,279] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-21 19:10:49,323] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-31T00:00:00+00:00 [queued]>
[2022-02-21 19:10:49,330] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-31T00:00:00+00:00 [queued]>
[2022-02-21 19:10:49,330] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 19:10:49,330] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-21 19:10:49,330] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 19:10:49,338] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_path> on 2022-01-31 00:00:00+00:00
[2022-02-21 19:10:49,341] {standard_task_runner.py:52} INFO - Started process 118 to run task
[2022-02-21 19:10:49,343] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_path', 'get_path', 'scheduled__2022-01-31T00:00:00+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmprmwa8vrr', '--error-file', '/tmp/tmpnhw8kxj5']
[2022-02-21 19:10:49,343] {standard_task_runner.py:77} INFO - Job 32: Subtask get_path
[2022-02-21 19:10:49,376] {logging_mixin.py:109} INFO - Running <TaskInstance: test_path.get_path scheduled__2022-01-31T00:00:00+00:00 [running]> on host 1594888d6c38
[2022-02-21 19:10:49,407] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_path
AIRFLOW_CTX_TASK_ID=get_path
AIRFLOW_CTX_EXECUTION_DATE=2022-01-31T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-31T00:00:00+00:00
[2022-02-21 19:10:49,407] {python.py:175} INFO - Done. Returned value was: ['/home/***/.local/bin', '/usr/local/lib/python37.zip', '/usr/local/lib/python3.7', '/usr/local/lib/python3.7/lib-dynload', '/home/***/.local/lib/python3.7/site-packages', '/usr/local/lib/python3.7/site-packages', '/opt/***/dags', '/opt/***/config', '/opt/***/plugins']
[2022-02-21 19:10:49,422] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_path, task_id=get_path, execution_date=20220131T000000, start_date=20220221T191049, end_date=20220221T191049
[2022-02-21 19:10:49,474] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-21 19:10:49,505] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
